package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@webscopeio/react-textarea-autocomplete",
-  "version": "4.6.3",
+  "version": "4.7.0",
```

Comment:
Can you please remove this change? I will do it during release process. I agree it should be 4.7.0 🙏